source/SpinalHDL/Simulation/bootstraps.rst

```diff
@@ -107,6 +116,45 @@ When in doubt about what to use, prefer ``doSim(...)``, for example :
        // Simulation code here
    }
 
+Here is a template where the simulation end by completting the simulation main thread execution :
+
+.. code-block:: scala
+
+    SimConfig.compile(new TopLevel).doSim { dut =>
+      SimTimeout(1000)
+      dut.clockDomain.forkStimulus(10)
+      dut.clockDomain.waitSamplingWhere(dut.counter.toInt == 20)
+      println("done")
+    }
+    
+Here is a template where the simulation end by explicitly calling a simSuccess() :
+
+.. code-block:: scala
+
+    SimConfig.compile(new TopLevel).doSimUntilVoid{ dut =>
+      SimTimeout(1000)
+      dut.clockDomain.forkStimulus(10)
+      fork {
+        dut.clockDomain.waitSamplingWhere(dut.counter.toInt == 20)
+        println("done")
+        simSuccess()
+      }
+    }
+
+Note is it equivalent to : 
```

Comment:
```
Note is it equivalent to:

```

Reply:
Missing empty line here provokes an error, which was reported but was ignored and the branch was merged. I think this merge was premature, and waiting for a review (even if you are the manager) would have avoided that.

See just above this title: https://spinalhdl.github.io/SpinalDoc-RTD/master/SpinalHDL/Simulation/bootstraps.html#running-multiple-tests-on-the-same-hardware